frontend/src/pages/read/pagePreviewThumbnails/PagePreviewThumbnails.tsx

```diff
@@ -26,34 +76,102 @@ export default function PagePreviewThumbnails({ pages }: { pages: Page[] }) {
       ? false
       : shown;
 
-  const items = Object.values(loadPages);
+  const firstReadingDirection = settings?.readingDirection
+    .split("-")?.[0]
+    .toLowerCase()!;
+  const navigationButtonsLocation = ["left", "top"].includes(
+    firstReadingDirection,
+  )
```

Comment:
I'm planning on changing the settings structure so this will cause issues in the future, consider simply testing
something like:
```ts
const navigationButtonsLocation = ["TOP-TO-BOTTOM", "LEFT-TO-RIGHT"].includes(settings?.readingDirection!);
```